cdap-examples/deploy_pom.xml

```diff
@@ -27,11 +27,19 @@
 
   <modules>
     <module>CountRandom</module>
```

Comment:
I think we should add a comment to the other list of example apps, pointing to this place? (so people know to add it to both places).